tests/test_sanity.py

```diff
@@ -114,6 +114,7 @@ def test_dotnet_trace(
     assert_collapsed: AssertInCollapsed,
     gprofiler_docker_image: Image,
     profiler_state: ProfilerState,
+    in_container: bool,
```

Comment:
Why added here? It is not used. This fixture is implied anyway by `application_pid`.